emscripten.py

```diff
@@ -753,7 +753,11 @@ def create_receiving(exports):
       for s in exports_that_are_not_initializers:
         mangled = asmjs_mangle(s)
         dynCallAssignment = ('dynCalls["' + s.replace('dynCall_', '') + '"] = ') if generate_dyncall_assignment and mangled.startswith('dynCall_') else ''
-        receiving += [dynCallAssignment + mangled + ' = asm["' + s + '"];']
+
+        if settings.MODULARIZE and settings.EXPORT_ALL:
+          receiving += [dynCallAssignment + mangled + ' = Module["{}"] = asm["{}"];'.format(mangled, s)]
+        else:
+          receiving += [dynCallAssignment + mangled + ' = asm["' + s + '"];']
```

Comment:
Maybe this could be
```
exportAssignment = f'Module["{mangled}"] =` if settings.MODULARIZE and settings.EXPORT_ALL else ''
receiving += [exportAssignment + dynCallAssignment + mangled + ' = asm["' + s + '"];']
```
to follow the same kind of pattern that `dynCallAssignment` uses?

Reply:
Yup good idea, I made a better version 👌